.eslintrc.cjs

```diff
@@ -135,7 +135,6 @@ module.exports = {
 
         // Temporarily disabled so we can inspect each warning one-by-one
         // TODO: Enable these.
-        '@typescript-eslint/no-unsafe-assignment': 'off',
```

Comment:
These are included in `@typescript-eslint/recommended-requiring-type-checking`. I'm a bit torn between explicitly enabling them in addition to the recommended set.

We should keep the recommended set enabled to make sure we get the new rules automatically. Perhaps it's also a good idea to explicitly enable the ones we care about to ensure they don't get disabled. What do you think?

Reply:
> These are included in `@typescript-eslint/recommended-requiring-type-checking`

Ah, that's the thing I was missing, good to know.
I think it would be a good idea to explicitly enable the most important of these, our list of enabled rules is already extensive and it will make it easier to see our original intention (without having to look up the defaults in the manual) the next time we will have a (potentially clueless) look at the config file.